docs/src/concrete-playback.md

```diff
@@ -0,0 +1,59 @@
+# Concretely playing back proof harnesses
+
+When the result of a certain check comes back as a `FAILURE`,
+this _experimental_ feature allows users to debug their proof harness and understand what
+caused the failing check.
+
+## Setup
+
+The user needs to have a fairly recent version of the Kani source code somewhere on their computer.
+To do this, run `git clone https://github.com/model-checking/kani.git`.
+Then, the user needs to add the following lines to their `Cargo.toml` file:
+```toml
+[dev-dependencies]
+kani = { path = "{path_to_kani}/library/kani", features = ["exe_trace"] }
+```
+
+## Usage
+
+Run Kani with the `--gen-exe-trace` flag.
+After verifying the proof harness checks, Kani will extract concrete values for the `kani::any()` variables
```

Comment:
Maybe describe what the user will see and not what the tool is doing under the hood. Say something like if there is a failure, Kani will print a test case to the console that includes concrete values that will trigger the failure concretely.

I would suggest that you add an example here of a failure and how the test will look like.

Reply:
Done